tajo-plan/src/main/java/org/apache/tajo/plan/util/PlannerUtil.java

```diff
@@ -133,11 +134,90 @@ public static boolean checkIfSimpleQuery(LogicalPlan plan) {
           }
         }
       }
+
+      if (!noWhere && scanNode.getTableDesc().isExternal() && scanNode.getTableDesc().getPartitionMethod() != null) {
```

Comment:
Ok. If so, would you add a TODO comment? 
It will be nice if it contains the specific issue number and description.